Source/Controllers/Preferences/Panes/SPFilePreferencePane.m

```diff
@@ -56,12 +60,18 @@ - (instancetype)init
 
 - (void)dealloc
 {
-	for(NSURL *url in resolvedBookmarks){
-		[url stopAccessingSecurityScopedResource];
-	}
+    SPLog(@"dealloc");
+    [SecureBookmarkManager.sharedInstance stopAllSecurityScopedAccess];
```

Comment:
Once again - concerned about stopping access while a bookmark is being used somewhere else.